src/cards/CommentCard.jsx

```diff
@@ -0,0 +1,15 @@
+function CommentCard({ children }) {
+  const { author, body, votes, comment_id, created_at } = children;
+  return (
+    <section>
+      <div>
+        <h2>{author}</h2>
+        <p>{body}</p>
+        <p>{created_at}</p>
```

Comment:
Would be better to have the timestamp formatted in a bit more of a human readable way, than the default. Have a think how you could manipulate the data yourself to do this, or look into some packages that can do it for you 😄